### ISSUE TITLE
Megadepth results

### ISSUE THREAD

**Issue by zwyking** [zju3dv/LoFTR#82]
Hi, I met a gap between val and test in Medadepth dataset. I recently use 4 gpus to train the model for 15 epochs. The map(@20) is  0.766 on val_list and 0.720 on test_list. I want to know if it is  normal?

**Comment by zehongs** [comment]
Is the val_img_size the same as the test_img_size?

**Comment by zehongs** [comment]
The DATASET.MGDPT_IMG_RESIZE in you data config might be different in the training and testing.

**Comment by zehongs** [comment]
Yes, you're right. In fact, most of the local feature extraction and matching methods are sensitive to the img_size.

**Comment by TruongKhang** [comment]
@zehongs do you remember the accuracy on the validation set when finishing the training on Megadepth? Can you please let me know (no need to be exactly)

**Comment by zehongs** [comment]
Hi, I thought we evaluated the test split directly, and the numbers converged quickly with low deviation. Not setting a validation split was not ideal though.

**Comment by TruongKhang** [comment]
Hi @zehongs, thank you. I tried to train the model. I got the AUC@10 is 66.1 on the evaluation set in the training phase. After that, I re-evaluated by running `test.py` in `scripts/reproduce_test/outdoor_ds.sh` and I got AUC@10 is 65.8. Is this small difference normal? Or they should be exactly the same one.

**Comment by zehongs** [comment]
They should be the same.
